app/src/test/java/net/wvffle/android/pb/schedule/BackendApiTest.java

```diff
@@ -16,4 +45,233 @@ public void testRoom() {
         assertEquals(hash, room.getHash());
         assertEquals("209", room.getName());
     }
+
+//    @Test
```

Comment:
Mozesz odkomentowac ten test?

Reply:
Nie bo ten z deserializerem ma taką samą nazwę chyba ze któremuś mam zmienić nazwę ?

Reply:
to wywal ten